src/ethereum/__init__.py

```diff
@@ -1,6 +1,17 @@
 """
 Ethereum Specification
 ^^^^^^^^^^^^^^^^^^^^^^
+Seeing as internet connections have been vastly expanding across the
+world, spreading information has become as cheap as ever. Bitcoin, for
+example, has demonstrated the possibility of creating a decentralized,
+and practically free, trade system that is accessible around the world.
```

Comment:
> and practically free

Is this still the case?

Reply:
It said that on the Yellow Paper but I may be wrong.

Reply:
good enough for me!